lms/static/styles/components/_ErrorDisplay.scss

```diff
@@ -11,7 +11,6 @@
 
   &__links {
     white-space: normal;
-    line-height: 1.25em;
```

Comment:
Since the font is smaller, this no longer word wraps, but even if it did, now the default font-size / line height is better matched so we don't need to override this.

Reply:
BTW, auditing for `line-height` and `font-size/line-height` combinations is on my longish mental TODO list for (S)CSS tech debt and pattern-clarity cleanup.